server/channels/app/status.go

```diff
@@ -98,7 +98,8 @@ func (a *App) SetCustomStatus(c request.CTX, userID string, cs *model.CustomStat
 		return err
 	}
 
-	user.SetCustomStatus(cs)
+	if err = user.SetCustomStatus(cs); err != nil {
+		c.Logger().Error("Failed to set custom status", mlog.String("userID", userID), mlog.Err(err))
```

Comment:
The code doesn't compile:

```
Error: channels/app/status.go:115:36: syntax error: unexpected request in argument list; possibly missing comma or )
Error: channels/app/status.go:116:12: syntax error: unexpected := in composite literal; possibly missing comma or }
```

Reply:
![image](https://github.com/user-attachments/assets/be7cff42-baca-43f3-a818-089347ea75c5)

The problem is from here. I don't see anything wrong with the argument list yet it flags a missing comma.

Reply:
You should take a look at the code that I highlighted. The error is there.

Reply:
Oh. Seen. The function was expecting a closing curly brace. My apologies.